### ISSUE TITLE
Bug: Import not working

### ISSUE THREAD

**Issue by wingkwong** [nocodb/nocodb#400]
**Steps To Reproduce**
Case 1: 
1. create a project (sqlite)
2. create some tables
3. export to zip
4. import it back to the same project. it shows "some internal error occurred".

```
Request URL: <REDACTED>/dashboard/?q=sqlOp_xcMetaTablesImportZipToLocalFsAndDb&project_id=test_AcEi
Request Method: POST
Status Code: 400 Bad Request
```

Response
```
{"msg":"Project with id 'project_1_Qd9-' already exist"}
```

Case 2:
1. create a project (sqlite)
2. create some tables
3. export to zip
4. Delete the project
5. import zip in My Projects. it shows "some internal error occurred".

```
Request URL: <REDACTED</dashboard/?q=sqlOp_xcMetaTablesImportZipToLocalFsAndDb
Request Method: POST
Status Code: 403 Forbidden
```

Response:
```
{"msg":"Not allowed"}
```

**Expected behavior**
Case 1: 

If it is not allowed to import to the same project. It may show some useful message instead of "some internal error occurred".

**Further discussion:**

if it is not allowed to import to the existing project. Then there is no point to include this import function in Project Metadata page. If the project does not exist, basically it will create a new one so it is weird to create a new project inside another project.

Case 2: metadata is expected to be loaded.

Node: **v12.22.1**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **pg**
ProjectOnRootDB: **false**
RootDB: **sqlite3**
PackageVersion: **0.11.6**

**Comment by pranavxc** [comment]
> Further discussion:
> 
> if it is not allowed to import to the existing project. Then there is no point to include this import function in Project Metadata page. If the project does not exist, basically it will create a new one so it is weird to create a new project inside another project.

We kept it because we are planning to allow import into same project as well since it may help roll back to a certain state. It's a bit complicated and we didn't implement it yet but soon we will so for time being we are just throwing the error.

May be import from zip we will hide for time being.

**Comment by pranavxc** [comment]
Fixed in `0.11.7`
